### PR TITLE
Add Ciena driver

### PR DESCRIPTION
**Pull request by jgroom33**
Also alphabetize the lists

### CODE REVIEW

File: ncclient/devices/ciena.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Ciena handler for device specific information.
+    """
+
+    def __init__(self, device_params):
+        super(CienaDeviceHandler, self).__init__(device_params)
+
+    def get_capabilities(self):
+        return [
+            "urn:ietf:params:netconf:base:1.0",
+        ]
```
Comment (einarnn) [inline ncclient/devices/ciena.py:16]: This has the effect of restricting what capabilities are offered by `ncclient` to a Ciena device. Is that your intention? If so, it is not actually necessary.

**Review by einarnn — COMMENTED** [review]
In general it's not clear to me what this device handler adds. A device handler is typically implemented if there is something special that is needed. For example, the IOS XE device handler adds a `save_config` operation:

https://github.com/ncclient/ncclient/blob/master/ncclient/devices/iosxe.py#L36-L39

This Ciena device handler doesn't seem to add any custom behaviour...?

File: ncclient/devices/ciena.py (hunk 1 above)
Comment (jgroom33) [reply ncclient/devices/ciena.py:16]: Thanks. I will remove.

**Comment by jgroom33** [comment]
No additional capabilities are required. The driver is necessary because the device does not support a namespace (nc) on the base namespace. So, this code is necessary:
def get_xml_base_namespace_dict(self):
        return {None: BASE_NS_1_0}

I attempted to use the default driver and override the namespace of the root, but could not figure it out. It seems the implementation of this library in Ansible provides limited ability to override any methods until some driver is specified.

**Comment by jgroom33** [comment]
Several of the drivers in this repo seem to require the namespace modification.
As an alternative to creating another vendor specific driver, I think there should be another generic driver that only has the namespace modification.
Do you think this is a better approach?

**Comment by jgroom33** [comment]
@einarnn

For reference, The following drivers implement the null namespace for BASE_NS_1_0:
* alu
* hpcomware
* ericsson
* huawei
* h3c
* sros
* huaweiyang

I'm not familiar with the other devices, but the only reason I opened this PR for the Ciena device is to null the BASE_NS_1_0 namespace. I suspect it might be similar for other vendors.

What I'm proposing is one of these options:
1. delete this PR. Open a new PR with a new driver `Generic_BASE_NS_1_0` that handles the namespace nullify
2. delete this PR. update the generic driver so it nullifies BASE_NS_1_0
3. Merge this PR

What would be your preference? I'm good with any of the above.

**Comment by einarnn** [comment]
@jgroom33, sorry for delay. I will merge this PR.

**Review by einarnn — APPROVED** [review]

**Pull request merged**